tasks/push_from_local.yml

```diff
@@ -15,6 +15,11 @@
     fail_msg: >-
       The path specified by `solr_core_conf_local_source` does not exist. Make sure the path is correct, and that the config files have been deployed to that directory.
 
+- name: Ensure the intermediate location starts clean
+  file:
+    state: absent
+    path: "{{ solr_core_conf_remote_intermediate }}/"
+
```

Comment:
@travis-bradbury the creation and removal of the intermediate dir will cause molecule's idempotency stage to fail. In that stage, no task is allowed to show as changed. Besides I don't think it's necessary anyway, since the [synchronize task specifies `delete`](https://github.com/AcroMedia/ansible-role-solr-core-config/blob/dba86a0c095aa2727444e4c92840048b9df39db2/tasks/push_from_local.yml#L31), which would purge any content which isn't a 1:1 match for the source dir.

Reply:
The only hitch to skipping cleanup would be if solr config might ever have sensitive info in it, but I don't think it does.